dev/src/dev.clj

```diff
@@ -89,10 +91,10 @@
   reset-changes!
   changes])
 
-(def initialized?
+(def initialized? "initialized?"
```

Comment:
we normally place the doc in a new line

Reply:
Yeah... but it looked so lonely. 😁 It's just clj-kondo triggered on me since I've touched dev.clj.